waku/factory/external_config.nim

```diff
@@ -134,10 +136,17 @@ type WakuNodeConf* = object
     ##  Application-level configuration
     protectedTopics* {.
       desc:
-        "Topics and its public key to be used for message validation, topic:pubkey. Argument may be repeated.",
-      defaultValue: newSeq[ProtectedTopic](0),
+        "Deprecated. Topics and its public key to be used for message validation, topic:pubkey. Argument may be repeated.",
```

Comment:
I'm sure you are aware of that: we will need to inform about that in CHANGELOG and then perform the actual removal in `v0.34.0`. Feel free to create an issue for the removal so that we don't forget it :)

Reply:
Good idea! Created the issue https://github.com/waku-org/nwaku/issues/2984 :)